addons/website_event_track/controllers/event_track.py

```diff
@@ -247,10 +250,7 @@ def _prepare_calendar_values(self, event):
         }
 
     def _event_agenda_get_tracks(self, event):
-        tracks_sudo = event.sudo().track_ids.filtered(lambda track: track.date)
-        if not request.env.user.has_group('event.group_event_manager'):
-            tracks_sudo = tracks_sudo.filtered(lambda track: track.is_published or track.stage_id.is_accepted)
-        return tracks_sudo
+        return event.sudo().track_ids.filtered(lambda track: track.date and track.is_visible)
```

Comment:
As discussed on "is_visible" field, would make condition explicit and remove "is_visible" field :) . Just to make things more explicit in controllers when dealing with a sudo() that bypasses ACLs.